microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -27,7 +31,7 @@ This release adds improvements related to the following components and concepts.
 //L3 major categories with features in each as L4s, for example:
 [id="microshift-4-14-rhel-edge"]
 === {op-system-ostree-first} {op-system-version}
-* {product-title} runs on {op-system-ostree} version {op-system-version} or later.
+* {product-title} runs on {op-system-ostree} version {op-system-version}.
 
```

Comment:
same as above - why "for Edge" only - it runs on RHEL 9.2, RHEL/Edge is not a different OS, its same RHEL 9.2, but in a different delivery format. I think we can simplify by just speaking of RHEL 9.2

Reply:
Non-ostree RHEL is not part of Red Hat Device Edge, is that correct? If so, we need to maintain a distinction. Not only for that combo, but also for updates, which are done differently and require the two listed separately.